bin/weekly_count.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+"""
+Get daily total issues reported on webcompat.
+"""
+
+import sys
+import datetime
+import json
+from urllib.parse import urljoin
+from urllib.request import Request
+from urllib.request import urlopen
+
+
+# Config
+SEARCH_URL = "https://api.github.com/search/"
+QUERY = "issues?q=repo:webcompat/web-bugs+created:{yesterday}"
+
+
+def get_remote_file(url):
+    """Request URL."""
+    req = Request(url)
+    req.add_header("User-agent", "webcompatMonitor")
+    req.add_header("Accept", "application/vnd.github.v3+json")
+    json_response = urlopen(req, timeout=240)
+    return json_response
+
+
+def get_issue_count(json_response):
+    """Get the number of issues (open or closed)."""
+    json_data = json.load(json_response)
+    if json_data["incomplete_results"] is False:
+        return json_data["total_count"]
+    else:
+        return None
+
+
+def get_yesterday(today_date):
+    """Get yesterday's date as a string from today's datetime object in UTC."""
+    one_day = datetime.timedelta(days=1)
+    return today_date - one_day
+
+
+def main():
+    """Core program."""
+    # Extract data from GitHub
+    yesterday = get_yesterday(datetime.datetime.now(datetime.timezone.utc))
```

Comment:
```suggestion
    today = datetime.date.today()
    yesterday = today - datetime.timedelta(days=1)
```
Then you could use directly `isoformat` method on it 

See

```python
>>> import datetime
>>> today = datetime.date.today()
>>> today
datetime.date(2019, 2, 1)
>>> today.isoformat()
'2019-02-01'
>>> yesterday = today - datetime.timedelta(days=1)
>>> yesterday
datetime.date(2019, 1, 31)
>>> yesterday.isoformat()
'2019-01-31'
```

Reply:
Yes, I considered using a date, rather than a datetime. It is less confusing, I think, without the time portion. (As you say below, we can always add one in manually if it turns out to be a problem.)

I had initially added it for ease of using the methods you and @magsout had already established, but it may not be necessary. (I was having problems extracting the iso-weekday with the dayjs library, but that was before I changed tactics.)